internal/services/instance/servers_data_source.go

```diff
@@ -183,15 +183,17 @@ func DataSourceInstanceServersRead(ctx context.Context, d *schema.ResourceData,
 		rawServer["zone"] = string(zone)
 		rawServer["name"] = server.Name
 		rawServer["boot_type"] = server.BootType
-		if server.Bootscript != nil {
-			rawServer["bootscript_id"] = server.Bootscript.ID
+		if server.Bootscript != nil { //nolint:staticcheck
```

Comment:
We need to add test and documentation to ensure that we detect when this field is removed from API responses and that the user are warned that the field is deprecated.